### PR TITLE
resolve merging issues.

### PR DESCRIPTION
**Pull request by dtao-oss**

### CODE REVIEW

File: maldoca/base/testing/test_utils.h [hunk 1]
```diff
@@ -22,6 +22,7 @@
 #include "maldoca/base/file.h"
 #include "maldoca/base/get_runfiles_dir.h"
 
+
```
Comment (b-maldoca) [inline maldoca/base/testing/test_utils.h:25]: remove empty line

File: maldoca/ole/dir_test.cc [hunk 2]
```diff
@@ -69,9 +69,11 @@ TEST(OLEDirectoryEntryTest, BasicTest) {
   EXPECT_TRUE(entry.IsInitialized());
 
   // Can't initialize twice.
-  ASSERT_DEATH(entry.Initialize("foo", DirectoryStorageType::Storage, 1, 2, 3,
+   ASSERT_DEATH(entry.Initialize("foo", DirectoryStorageType::Storage, 1, 2, 3,
```
Comment (b-maldoca) [inline maldoca/ole/dir_test.cc:72]: remove space

File: maldoca/service/common/office_processing_component_test.cc [hunk 3]
```diff
@@ -94,8 +96,12 @@ void ValidateParsedProto(absl::string_view file_base, absl::string_view ext,
                          const ProcessorConfig& config) {
   std::string input_file_name = absl::StrCat(file_base, ".", ext);
   std::string input;
+<<<<<<< HEAD
```
Comment (b-maldoca) [inline maldoca/service/common/office_processing_component_test.cc:99]: seems there r some merge issues, no?

File: maldoca/ole/dir_test.cc (hunk 2 above)
Comment (dtao-oss) [reply maldoca/ole/dir_test.cc:72]: fixed

File: maldoca/service/common/office_processing_component_test.cc (hunk 3 above)
Comment (dtao-oss) [reply maldoca/service/common/office_processing_component_test.cc:99]: fixed.

File: maldoca/base/testing/test_utils.h (hunk 1 above)
Comment (dtao-oss) [reply maldoca/base/testing/test_utils.h:25]: fixed

File: maldoca/ole/dir_test.cc [hunk 4]
```diff
@@ -72,6 +72,8 @@ TEST(OLEDirectoryEntryTest, BasicTest) {
   ASSERT_DEATH(entry.Initialize("foo", DirectoryStorageType::Storage, 1, 2, 3,
                                 4, 5, 6, clsid, 8, 9, 10),
                "Check failed: !is_initialized_");
+  /*entry.Initialize("foo", DirectoryStorageType::Storage, 1, 2, 3, 4, 5, 6,
```
Comment (b-maldoca) [inline maldoca/ole/dir_test.cc:75]: can u remove this one?

**Review by b-maldoca — COMMENTED** [review]
also clang-format complains

**Review by b-maldoca — APPROVED** [review]

File: maldoca/ole/dir_test.cc (hunk 4 above)
Comment (dtao-oss) [reply maldoca/ole/dir_test.cc:75]: done

**Pull request merged**